etcdserver/etcdhttp/http.go

```diff
@@ -29,10 +29,7 @@ import (
 
 const (
 	// time to wait for response from EtcdServer requests
-	// 5s for disk and network delay + 10*heartbeat for commit and possible
-	// leader switch
-	// TODO: use heartbeat set in etcdserver
-	defaultServerTimeout = 5*time.Second + 10*(100*time.Millisecond)
+	defaultServerTimeout = time.Duration(math.MaxInt64)
```

Comment:
We need some default value for server timeout. It is mostly left for customized timeout i think.